merlin/celery.py

```diff
@@ -54,65 +54,46 @@
 results_ssl = False
 try:
     BROKER_URI = broker.get_connection_string()
-    LOG.info(f"broker: {broker.get_connection_string(include_password=False)}")
+    LOG.debug(f"broker: {broker.get_connection_string(include_password=False)}")
     broker_ssl = broker.get_ssl_config()
-    LOG.info(f"broker_ssl = {broker_ssl}")
+    LOG.debug(f"broker_ssl = {broker_ssl}")
     RESULTS_BACKEND_URI = results_backend.get_connection_string()
     results_ssl = results_backend.get_ssl_config(celery_check=True)
-    LOG.info(
+    LOG.debug(
         f"results: {results_backend.get_connection_string(include_password=False)}"
     )
-    LOG.info(f"results: redis_backed_use_ssl = {results_ssl}")
+    LOG.debug(f"results: redis_backed_use_ssl = {results_ssl}")
 except ValueError:
     # These variables won't be set if running with '--local'.
     BROKER_URI = None
     RESULTS_BACKEND_URI = None
 
+app = Celery("merlin")
 
-app = Celery(
-    "merlin",
+# load merlin config defaults
+app.conf.update(**celeryconfig.DICT)
+
+# load config overrides from app.yaml
+if (
```

Comment:
Do we need to say anything if there is no override? How about just else part.

Reply:
else pass? Sure, I can switch the no override log message to debug mode.

Reply:
I just mean change the logic of the if to only print if there is an override.